### DISCUSSION TITLE
New game recommend---Zenless Zone Zero

### DISCUSSION THREAD

**Discussion by jiashuaijieshuo** [share121/inter-knot#120] [其他]
"Zenless Zone Zero" is a captivating urban exploration game crafted by the innovative developers at miHoYo. In this immersive experience, players assume the unique identity of "Rope Craftsmen," tasked with navigating the enigmatic "Void" alongside a cast of characters. Following a series of rigorous beta tests, the game made its official debut in July 2024. The game's narrative unfolds in New Elysium, a bustling metropolis that serves as the focal point for numerous individuals driven by diverse motivations to delve into the Void's secrets. Players are engaged in aiding these characters in accomplishing their quests, all the while unraveling the rich tapestry of their personal narratives. To enhance the gaming experience, "Zenless Zone Zero" provides an array of welfare events designed to reward and incentivize player participation. 
![img1306669991_1722516804243](https://github.com/user-attachments/assets/8d016f26-45c1-4324-abbc-873e859fb1db)

**Comment by silverwolf510trillion** [comment]
very good picture, love from New Eridu.

**Comment by lllliufeng** [comment]
wow

**Comment by GanlinSama** [comment]
Well,why you're all speaking Eng?this is a Chinese forum,isn't it?

**Reply by jiashuaijieshuo** [reply]
我最开始不知道还支持中文

**Comment by Y-0930** [comment]
Oh my god，where am i

**Comment by Y084** [comment]
You all speak English, I can't understand it.

**Reply by jiashuaijieshuo** [reply]
对不起，我刚来，不清楚这玩意支不支持中文

**Comment by Iincan** [comment]
haha what can i say

**Comment by user1-cloud** [comment]
maybe you are right.but i cant understand why mihoyo is called "film and iron" company; why Proxy is called "rope maker",and why people want to explore "void".it seems like mihoyo and zzz have some pretty weird concepts.

**Comment by CW10086114404** [comment]
Wow Gawr Gura and Ellen Joe I all like 😍

**Comment by Rolmans** [comment]
Oh,God.Chinese!Chinese!

**Comment by jiashuaijieshuo** [comment]
Wow!So amazing
![b70a872e21bae0bfae3642193df79327](https://github.com/user-attachments/assets/01ed4792-cca8-4bfc-906d-02eddb4e306e)

**Comment by jiashuaijieshuo** [comment]
Good👍
![img-1948839571_1722515127488](https://github.com/user-attachments/assets/66f6ff3b-89e5-4412-930d-1e6ee980f3f7)

**Comment by jiashuaijieshuo** [comment]
![1723271782729](https://github.com/user-attachments/assets/6b17dcf7-23c0-41e5-9ab0-275e96e1f33c)
![1723216042373](https://github.com/user-attachments/assets/45c7a8f6-1d31-4cbd-b3ee-140339891d98)
![IMG_20240809_230935](https://github.com/user-attachments/assets/19267523-c051-47e6-a160-40ec6f357938)
![IMG_20240809_231013](https://github.com/user-attachments/assets/294f3c1b-2138-42b0-ae46-93a43cba1f43)

**Comment by yty2003** [comment]
不懂英语

**Comment by jiashuaijieshuo** [comment]
6

________________________________
发件人: yty2003 ***@***.***>
发送时间: 2024年8月13日 12:41
收件人: share121/inter-knot ***@***.***>
抄送: 贾帅 ***@***.***>; Author ***@***.***>
主题: Re: [share121/inter-knot] New game recommend---Zenless Zone Zero (Discussion #120)


不懂英语

―

**Comment by LING-JI-ZZZ** [comment]
乐<img src="https://img-static.mihoyo.com/communityweb/upload/460474087917ad69a4ff975a3132a0df.png" alt="妮可 哈哈" width="75" >

**Comment by haifanhen6** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/04/28/b71f9bae78003f98a47f0d6e0765324d_7897172668253324041.png" alt="米仔_休息" width="75" >

**Comment by HifumiMizuhara** [comment]
very good post, love from Inazuma city

**Comment by HauUhang** [comment]
SO nice！

**Comment by CloudSwordSage** [comment]
ummm, I need English2Chinese<img src="https://img-static.mihoyo.com/communityweb/upload/d57ac9b36df80b48b1acb5769ac349ac.png" alt="米游兔可怜兮兮" width="75" >
